### DISCUSSION TITLE
is there a way to invite someone to my organization without email?

### DISCUSSION THREAD

**Discussion by brucemackenzie** [dani-garcia/vaultwarden#1927] [Q&A]
I setup VaultWarden and LOVE it. So much easier to use than Keepass. Thank you to the community for making this happen!

I'm trying to get my wife migrated and it seems we need an "organization" to make that happen because we share a lot of passwords.
I had no problem creating an organization, but now I need to invite her.
That seems to require that I somehow setup an email server?
I assume that is a very complex undertaking (from some of the threads on here) and that most email systems are justifiably suspicious of email from unknown servers like my own.
Is there a way to bypass all of that hassle and just copy/paste an invitation to her?
Some other method?
Sorry if this has been asked - couldn't find it in my searches.

**Comment by jjlin** [comment]
It's not very hard to set up an external email service. There are a few examples with free tiers at

https://github.com/dani-garcia/vaultwarden/wiki/SMTP-configuration#smtp-servers
